src/components/_global/BalAsset/BalAssetSet.vue

```diff
@@ -55,3 +56,16 @@ export default defineComponent({
   }
 });
 </script>
+
+<style scoped>
+.token-set {
+  @apply absolute;
+  box-shadow: 0 0 0 1.5px #ffffff;
```

Comment:
imo it looks better without these borders, it's already quite intricate visually, I feel like the borders over complicate it.

Reply:
Creating separation like this is a fairly common design pattern when overlapping images. It helps people to visually distinguish the different tokens. 

For example, see how Instagram does this within their mobile app on overlapping images which are even smaller than our token logos.

![IMG_B3DACB92D53B-1](https://user-images.githubusercontent.com/1121139/125497445-f4e3d1e3-fc34-4600-8f12-ef8906e34e3c.jpeg)


![IMG_EF8D69887255-1 2](https://user-images.githubusercontent.com/1121139/125497458-c0fff21f-08f4-498a-91cb-3f8b450c4503.jpeg)

Reply:
Ahh yeah this is makes more sense now, but we should be using the tailwind colors and also adjusting those colors on row hover so it matches the hover background color. If you need help with that lmk.

Reply:
Done! I initially preferred to use a box-shadow instead of a border to retain the with and height of the icons (instead of losing a couple of pixels to an inner border... Anyway, it should be good now.

Reply:
hmm I think this is only useful if you don't use shadows under the icons. See below:

<img width="407" alt="Screenshot 2021-07-14 at 10 03 55" src="https://user-images.githubusercontent.com/2406506/125595332-f093e46d-6928-465f-b1fe-34fc7b7b5767.png">

With shadows it doesn't look like they're cut out of each other like it does on instagram, I think this is what I was noticing. I would suggest we either we have shadows with no borders or borders with no shadows, and if we go for borders then I'd probabl increase the width to make it more distinctive:

<img width="367" alt="Screenshot 2021-07-14 at 10 08 32" src="https://user-images.githubusercontent.com/2406506/125595828-92af7c36-a1bf-4c90-9679-b024a8737418.png">

Reply:
Happy to remove the box-shadows.

I agree with increasing the borders, but the problem is that increasing the border size reduces the token image size since css borders are inner borders. This is why I was initially using box-shadows to act as outer borders but then we lose tailwind's color system... Maybe we could increase the size of the token images here by 2px, and then have `border-2` set on them?